README

```diff
@@ -74,27 +74,49 @@ Curl Examples:
 If the UI is not your thing, curl probably is! Here's an example of using
 curl to create a gateway node.
 
-curl --user admin:admin -d ip_address=192.168.122.14 \
+$ curl --user admin:admin -d ip_address=192.168.122.14 \
      -X PUT http://192.168.122.14:5000/api/gateway/iqn.2003-01.com.redhat.iscsi-gw:ceph-gw1/rh7-gw2
 
 IPv6 Support:
 Make sure the IPv6 addresses used are global unicast ones, and then the
 URL link could be:
 
-curl --user admin:admin -d ip_address=2006:ac81::1104 \
+$ curl --user admin:admin -d ip_address=2006:ac81::1104 \
      -X PUT http://[2006:ac81::1104]:5000/api/gateway/iqn.2003-01.com.redhat.iscsi-gw:ceph-gw2/rh7-gw2
 
 And mixing the IPv4 and IPv6 addresses is also allowed as below:
 
-curl --user admin:admin -d ip_address=192.168.122.14 \
+$ curl --user admin:admin -d ip_address=192.168.122.14 \
      -X PUT http://[2006:ac81::1104]:5000/api/gateway/iqn.2003-01.com.redhat.iscsi-gw:ceph-gw1/rh7-gw2
 
-curl --user admin:admin -d ip_address=2006:ac81::1104 \
+$ curl --user admin:admin -d ip_address=2006:ac81::1104 \
      -X PUT http://192.168.122.14:5000/api/gateway/iqn.2003-01.com.redhat.iscsi-gw:ceph-gw2/rh7-gw2
 
 NOTE: please make sure both the IPv4 and IPv6 addresses are in the trusted
 ip list in iscsi-gateway.cfg.
 
+Mutual CHAP Authentication:
+
+In the initiator node, uncomment and set the following options in /etc/iscsi/iscsid.conf:
+
+node.session.auth.authmethod = CHAP
+node.session.auth.username = ${admin_username}
+node.session.auth.password = ${admin_password}
+node.session.auth.username_in = ${mutual_username}
+node.session.auth.password_in = ${mutual_password}
```

Comment:
I think this should be added to the Ceph docs where the CHAP authentication is already touched on, so that it stays in one place.  Do you mind if I do that based on this PR?

Reply:
Sounds good, then let's paste that link in the `README` here ? At least we know where to find that doc and how to set it.

Reply:
https://github.com/ceph/ceph/pull/46426

Reply:
> Sounds good, then let's paste that link in the `README` here ? At least we know where to find that doc and how to set it.

It is already there, at the bottom.